src/python_testing/TC_PAVST_2_4.py

```diff
@@ -0,0 +1,307 @@
+            )
+            asserts.assert_greater_equal(
+                len(aAllocatedVideoStreams),
+                1,
+                "AllocatedVideoStreams must not be empty",
+            )
+
+            await self.send_single_cmd(
+                cmd=Clusters.CameraAvStreamManagement.Commands.AudioStreamAllocate(
+                    streamUsage=0,
+                    audioCodec=0,
+                    channelCount=2,
+                    sampleRate=48000,
+                    bitRate=96000,
+                    bitDepth=16,
+                ),
+                endpoint=endpoint,
+            )
+
+            aAllocatedAudioStreams = await self.read_single_attribute_check_success(
+                endpoint=endpoint,
+                cluster=avcluster,
+                attribute=avattr.AllocatedAudioStreams,
+            )
+            asserts.assert_greater_equal(
+                len(aAllocatedAudioStreams),
+                1,
+                "AllocatedAudioStreams must not be empty",
+            )
+
+            await self.send_single_cmd(
+                cmd=pvcluster.Commands.AllocatePushTransport(
+                    {
+                        "streamUsage": 0,
+                        "videoStreamID": 1,
+                        "audioStreamID": 1,
+                        "endpointID": 1,
+                        "url": "https://localhost:1234/streams/1",
+                        "triggerOptions": {"triggerType": 2},
+                        "ingestMethod": 0,
+                        "containerFormat": 0,
+                        "containerOptions": {
+                            "containerType": 0,
+                            "CMAFContainerOptions": {"chunkDuration": 4},
+                        },
+                        "expiryTime": 5,
+                    }
+                ),
+                endpoint=endpoint,
+            )
+
+        self.step(2)
+        if self.pics_guard(self.check_pics("PAVST.S.A0001")):
+            transport_configs = await self.read_single_attribute_check_success(
+                endpoint=endpoint,
+                cluster=pvcluster,
+                attribute=pvattr.CurrentConnections,
+            )
+            asserts.assert_greater_equal(
+                len(transport_configs), 1, "TransportConfigurations must not be empty!"
+            )
+            aTransportOptions = transport_configs[0].TransportOptions
+            aConnectionID = transport_configs[0].ConnectionID
+
+        # TH1 sends command
+        self.step(3)
+        if self.pics_guard(self.check_pics("PAVST.S.A0001")):
+            status = await self.send_single_cmd(
+                cmd=pvcluster.Commands.ModifyPushTransport(
+                    {
+                        "connectionID": 10,
+                    }
+                ),
+                endpoint=endpoint,
+            )
+            asserts.assert_true(
+                status == pvattr.Status.NOT_FOUND,
```

Comment:
NOT_FOUND isn't a Cluster defined status code.  Shouldn't this be Status.NotFound?